.github/workflows/validate-github-access.yml

```diff
@@ -0,0 +1,313 @@
+name: Validate GitHub Access Token
+
+on:
+  workflow_dispatch:
+    inputs:
+      github_base_url:
+        description: 'GitHub Base URL (use https://github.com for GitHub.com or your GitHub Enterprise URL)'
+        required: true
+        default: 'https://github.com'
+        type: string
+      github_org:
+        description: 'Organization name where your repositories are located'
+        required: true
+        default: 'gruntwork-io'
+        type: string
+      test_repo:
+        description: 'Repository name to test access (should be accessible with your token)'
+        required: true
+        default: 'patcher-cli'
+        type: string
+      test_version:
+        description: 'Release version to test (optional, defaults to latest)'
+        required: false
+        default: 'v0.15.2'
+        type: string
+      github_token_secret:
+        description: 'Name of the secret containing your GitHub token (defaults to GITHUB_TOKEN)'
+        required: false
+        default: 'GITHUB_TOKEN'
+        type: string
+
+jobs:
+  validate-access:
+    name: Validate GitHub Access
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Validate Inputs
+        run: |
+          echo "🔍 Validating GitHub Access Token"
+          echo "=================================="
+          echo "GitHub Base URL: ${{ inputs.github_base_url }}"
```

Comment:
NIT: You should avoid interpolating these values directly into the `run` like this. It's better to use an `env`, and interpolate the values using an environment variable.

e.g.

```
    steps:
      - name: Validate Inputs
        env:
          GITHUB_BASE_URL: ${{ inputs.github_base_url }}
        run: |
          echo "🔍 Validating GitHub Access Token"
          echo "=================================="
          echo "GitHub Base URL: ${GITHUB_BASE_URL}"
```

This prevents issues related to shell injection and interpolation errors.

Reply:
Updated in 796c5b1588d3da5b8c7072c0909dfccf1b35941c